gradle.properties

```diff
@@ -69,4 +69,4 @@ signing.keyId=
 # This will be defaulted to ~/.gnupg/secring.gpg if not provided as a command line property
 signing.secretKeyRingFile=
 # The version of Java to build the 'client' project and all its dependencies in.
-clientJavaVersion=8
\ No newline at end of file
+clientJavaVersion=8
```

Comment:
Why has this line changed? This change seems to be unrelated to the issue at hand.

Reply:
I've fixed it. Removed the extra space/ line.